src/main/kotlin/ro/hibyte/betting/dto/EventResponse.kt

```diff
@@ -0,0 +1,22 @@
+package ro.hibyte.betting.dto
+
+import ro.hibyte.betting.entity.Status
+import java.time.Instant
+
+data class EventResponse(
```

Comment:
Let me know when you get to this, we need to talk about what these classes are for